### ISSUE TITLE
VSCode's own Settings Sync

### ISSUE THREAD

**Issue by ackvf** [shanalikhan/code-settings-sync#1247]
Now that VSCode has its own Settings Sync capability https://code.visualstudio.com/updates/v1_48#_settings-sync, how does this one compares to it and what is the recommended approach by you, the author? Does it make your extension obsolete?

_(PS: Slack is blocked on my company's laptop)_

**Comment by Fasani** [comment]
Also when I try to use your extension with the new GitHub codespaces I get told this, (wondering if this could work there too)
**Can not install 'Settings Sync' because this extension has defined that it cannot run on the remote server.**

**Comment by shanalikhan** [comment]
Duplicate of #1188
